coding-the-legislation/reforms.md

```diff
@@ -1,104 +1,94 @@
+### Parametric reforms
+
+Unlike *structural reforms*, *parametric reforms* do not directly modify any formula: they apply changes to the legislation parameters.
 
-Here is an example:
+To modify the legislation parameters in the reform, you can call the method `self.modify_legislation_json`, which takes a function as a parameter.
+
+This function defines the modifications you want to apply to the legislation. It takes as a parameter a copy of the reference tax and benefit system `legislation_json`. You can then modify and return this `llegislation_json`.
+
+For instance:
 
 ```python
 def modify_legislation_json(reference_legislation_json_copy):
     reference_legislation_json_copy['children']['xxx']['values'][0]['value'] = 0
     return reference_legislation_json_copy
 
 
-def build_reform(tax_benefit_system):
-    Reform = reforms.make_reform(
-        key = 'new_legislation',
-        name = u'Dummy reform with new legislation',
-        reference = tax_benefit_system,
-        )
+class some_reform(Reform):
+    name = u'Change some legislation parameters'
+
+    def apply(self):
+        self.modify_legislation_json(modifier_function = modify_legislation_json)
+```
+
+> Note that you have to know about the data structure of the legislation JSON to modify it.
+
+## Using a reform in python
+
+Reforms can be applied in python with the following syntax:
+
+```py
+from openfica_some_country import CountryTaxBenefitSystem
+from some_other_package import some_reform
 
-    reform = Reform()
-    reform.modify_legislation_json(modifier_function = modify_legislation_json)
-    return reform
+tax_benefit_system = CountryTaxBenefitSystem()
+
+reformed_tax_benefit_system = some_reform(tax_benefit_system)
 ```
 
-> You have to know about the structure of the legislation JSON data structure to modify it.
+Reforms can be chained:
+
+```py
+from some_package import some_reform
+from some_other_package import some_other_reform
+
+reformed_tax_benefit_system = some_other_reform(
+    some_reform(tax_benefit_system)
+    )
+```
 
-For more details see the examples linked in the tl;dr section above.
+> The [Getting_Started Notebook](https://github.com/openfisca/openfisca-france/blob/master/notebooks/getting-started.ipynb) contains an example of reform use.
 
-## Use from the Web API
+## Using a reform from the Web API
 
 Please read the dedicated documentation:
-[OpenFisca-Web-API reforms](https://github.com/openfisca/openfisca-web-api/tree/next/docs/reforms.md)
+[OpenFisca-Web-API reforms](../openfisca-web-api/reforms.html)
 
 ## Real examples
 
-Examples can be found on the [community page](../../community.html).
+Examples can be found on the [community page](../../community.html), as well as on the [OpenFisca-France reforms directory](https://github.com/openfisca/openfisca-france/tree/master/openfisca_france/reforms).
```

Comment:
There is only one example on `community`, what about linking directly to https://github.com/openfisca/openfisca-france-extension-revenu-de-base ?